src/ui/buttons/styles/responsive.js

```diff
@@ -22,10 +22,13 @@ export function buttonResponsiveStyle({ height, fundingEligibility } : {| height
         const minDualWidth = Math.max(Math.round(buttonHeight * BUTTON_MIN_ASPECT_RATIO * (100 / WALLET_BUTTON_PERC)), MIN_SPLIT_BUTTON_WIDTH);
 
         const { paylater } = fundingEligibility;
-        const isInGerman = paylater?.products?.paylater?.variant === 'DE';
+        const resizeLabel =
```

Comment:
Could we prefix this boolean variable name with `is`, `has`, or `should`?
```suggestion
        const shouldResizeLabel =
```

Reply:
Done!